src/client/app/translations/data.ts

```diff
@@ -243,6 +243,7 @@ const LocaleTranslationData = {
 		"help.home.compare.sort.tip": "Allows user to select the order of multiple comparison graphs to be Alphabetical (by name), Ascending (greatest to least reduction in usage) and Descending (least to greatest reduction in usage).  Please see {link} for further details and information.",
 		"help.home.error.bar": "Toggle error bars with min and max value. Please visit {link} for further details and information.",
 		"help.home.export.graph.data": "With the \"Export graph data\" button, one can export the data for the graph when viewing either a line or bar graphic. The zoom and scroll feature on the line graph allows you to control the time frame of the data exported. The \"Export graph data\" button gives the data points for the graph and not the original meter data. The \"Export graph meter data\" gives the underlying meter data (line graphs only). Please visit {link} for further details and information on when meter data export is allowed.",
+		"help.home.history": "Allows the user to navigate through the recent history of graphs.",
```

Comment:
Could the end of the help have the standard "Please visit {link} for further details and information." at the end. In the other languages too. This will link to the correct (coming) page with the other change.

Reply:
done